NachoSpigot-Server/pom.xml

```diff
@@ -80,59 +80,60 @@
         <dependency>
             <groupId>net.sf.jopt-simple</groupId>
             <artifactId>jopt-simple</artifactId>
-            <version>3.2</version>
+            <version>5.0.4</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>jline</groupId>
             <artifactId>jline</artifactId>
-            <version>2.12.1</version>
+            <version>2.14.6</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>org.xerial</groupId>
             <artifactId>sqlite-jdbc</artifactId>
-            <version>3.7.2</version>
+            <version>3.36.0.3</version>
```

Comment:
Same like with the mysql connector

Reply:
I get the mysql connector part but this one is kinda ridiculous because version 3.7.2 is literally from 2010

Reply:
we know :/ updating yaml, mysql and sqlite stuff causes always issues of that badly old coded plugins

Reply:
> I get the mysql connector part but this one is kinda ridiculous because version 3.7.2 is literally from 2010

When you create plugins based on a driver of 2010, that's why you can't just bump it without testing with some common plugins

Reply:
#176